pulsar/managers/util/job_script/CLUSTER_SLOTS_STATEMENT.sh

```diff
@@ -26,6 +26,19 @@ elif [ -n "$LSB_DJOB_NUMPROC" ]; then
 elif [ -n "$GALAXY_SLOTS" ]; then
     # kubernetes runner injects GALAXY_SLOTS into environment
     GALAXY_SLOTS=$GALAXY_SLOTS
+elif [ -n "$ROOT_MAX_THREADS" ]; then
+    # HTCondor sets a couple of environment variables read by various
```

Comment:
This is part of every job script now, I think it's fine to skip the comment. I would somewhat prefer to use `PYTHON_CPU_COUNT`, I think more people know about python than root (TIL)